src/client/src/ui/analytics/epics/index.ts

```diff
@@ -1,11 +1,15 @@
 import { combineEpics } from 'redux-observable'
 import { publishPositionUpdateEpic } from './analyticsServiceEpic'
 import { analyticsServiceEpic } from './epics'
+import { InteropServices, PlatformAdapter } from 'rt-components'
 
-const epics = [analyticsServiceEpic]
+export default ({ platform }: { platform: PlatformAdapter }) => {
```

Comment:
It seems that the type of the parmater is `ApplicationDependencies` and you should use it accordingly

Reply:
Included that type now